src/sc-main.c

```diff
@@ -54,19 +54,19 @@ int sc_main(int argc, char **argv)
 	if (argc < NR_ARGS + 1)
 		die("Usage: %s <security-tag> <binary>", argv[0]);
 
-	const char *appname = argv[1];
-	debug("appname is %s", appname);
+	const char *executable_name = argv[1];
```

Comment:
If you are changing this, can you just call it as such, and remove `aa_profile` entirely as it was always the same thing